util/set/set.go

```diff
@@ -7,9 +7,33 @@ package set
 // Set is a set of T.
 type Set[T comparable] map[T]struct{}
 
+// NewFromSlice returns a new set constructed from the elements in slice.
+func NewFromSlice[T comparable](slice []T) Set[T] {
```

Comment:
s/NewFromSlice/SetOf/ ? ala views.SliceOf or reflect.TypeOf/ValueOf? Then it avoids "New" again.

Reply:
Oh yeah, good idea. Coming up. Thanks!